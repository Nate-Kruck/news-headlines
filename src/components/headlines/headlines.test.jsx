import React from 'react';
import { render, cleanup } from '@testing-library/react';
import TopHeadlines from './headlines';

describe('TopHeadLines component', () => {
  afterEach(() => cleanup());
  it('renders TopHeadlines', () => {
    const { asFragment } = render(<TopHeadlines
      headlines={[
        {
          sourceId: 'cnn',
          sourceName: 'CNN',
          description: 'some text',
          sourceURL: 'http://placeholder.com/headlines'
        }
      ]}
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
