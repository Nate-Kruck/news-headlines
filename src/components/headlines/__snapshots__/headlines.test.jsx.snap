// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopHeadLines component renders TopHeadlines 1`] = `
<DocumentFragment>
  <ul
    data-tesid="headlines"
  >
    <li />
  </ul>
</DocumentFragment>
`;
